integration-tests/vm.install

```diff
@@ -1,9 +1,7 @@
 #!/bin/sh
 set -eux
 
-if [ ! -f /usr/libexec/cockpit-ws ]; then
-    dnf install -y cockpit
-fi
+rpm -q cockpit-ws || dnf install -y cockpit-ws
```

Comment:
can't we simply unconditionally install the package? won't the dnf command be a no-op and succeed if it's already installed?

Reply:
Yes, I thought about this as well, I am not really sure why it's split that way. If you don't know a reason either, I'll just do that.

Reply:
dunno, it was introduced with commit 23e6c12fbd367736457f733782da018808f04add by @mvollmer - Markus, was there a reason to check for a file of cockpit-was rather than installing it directly?

@martinpitt in case, I'd unify this with the `dnf install` command that is few lines below